test/integration/go.mod

```diff
@@ -3,51 +3,59 @@ module github.com/yahoo/k8s-athenz-istio-auth/test/integration
 go 1.12
 
 require (
+	github.com/MakeNowJust/heredoc v0.0.0-20171113091838-e9091a26100e // indirect
+	github.com/NYTimes/gziphandler v1.1.1 // indirect
 	github.com/ardielle/ardielle-go v1.5.2
-	github.com/aws/aws-sdk-go v1.23.2 // indirect
-	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/armon/go-metrics v0.0.0-20190430140413-ec5e00d3c878 // indirect
+	github.com/armon/go-radix v1.0.0 // indirect
+	github.com/chai2010/gettext-go v0.0.0-20170215093142-bf70f2a70fb1 // indirect
+	github.com/coreos/bbolt v1.3.3 // indirect
 	github.com/coreos/etcd v3.3.15+incompatible
-	github.com/coreos/go-systemd v0.0.0-20190719114852-fd7a80b32e1f // indirect
-	github.com/d2g/dhcp4client v1.0.0 // indirect
-	github.com/emicklei/go-restful-swagger12 v0.0.0-20170926063155-7524189396c6 // indirect
-	github.com/go-openapi/strfmt v0.19.2 // indirect
-	github.com/grpc-ecosystem/grpc-gateway v1.9.5 // indirect
-	github.com/kardianos/osext v0.0.0-20190222173326-2bc1f35cddc0 // indirect
-	github.com/pquerna/cachecontrol v0.0.0-20180517163645-1555304b9b35 // indirect
-	github.com/prometheus/procfs v0.0.3 // indirect
-	github.com/robfig/cron v1.2.0 // indirect
-	github.com/stretchr/testify v1.3.0
-	github.com/vmware/govmomi v0.21.0 // indirect
-	github.com/xanzy/go-cloudstack v2.4.1+incompatible // indirect
-	github.com/yahoo/athenz v1.8.23
+	github.com/hashicorp/go-hclog v0.9.0 // indirect
+	github.com/konsorten/go-windows-terminal-sequences v1.0.3 // indirect
+	github.com/sirupsen/logrus v1.8.1
+	github.com/stretchr/testify v1.5.1
+	github.com/yahoo/athenz v1.9.30
 	github.com/yahoo/k8s-athenz-istio-auth v0.0.0-00010101000000-000000000000
-	github.com/yahoo/k8s-athenz-syncer v0.1.1
-	go.etcd.io/etcd v3.3.13+incompatible // indirect
-	gopkg.in/gcfg.v1 v1.2.3 // indirect
-	gopkg.in/warnings.v0 v0.1.2 // indirect
-	istio.io/api v0.0.0-20190906131201-ca4ba7013e1c
-	istio.io/istio v0.0.0-20190911205955-c2bd59595ce6
-	k8s.io/api v0.0.0-20190905160310-fb749d2f1064
-	k8s.io/apiextensions-apiserver v0.0.0-20190906235842-a644246473f1
-	k8s.io/apimachinery v0.0.0-20190831074630-461753078381
+	github.com/yahoo/k8s-athenz-syncer v0.1.7
+	go.etcd.io/etcd v3.3.25+incompatible
+	istio.io/api v0.0.0-20200513175333-ae3da0d240e3
+	istio.io/client-go v0.0.0-20200513180646-f8d9d8ff84e6
+	istio.io/istio v0.0.0-20200708165503-80f49905d910
+	istio.io/pkg v0.0.0-20200324191837-25e6bb9cf135
+	k8s.io/api v0.17.14
+	k8s.io/apiextensions-apiserver v0.17.14
+	k8s.io/apimachinery v0.17.14
 	k8s.io/client-go v11.0.0+incompatible
```

Comment:
can we upgrade client-go version as well?

Reply:
Done